src/test/streams_tests.cpp

```diff
@@ -110,6 +110,9 @@ BOOST_AUTO_TEST_CASE(streams_vector_reader)
     // Reading after end of byte vector throws an error even if the reader is
     // not totally empty.
     BOOST_CHECK_THROW(new_reader >> d, std::ios_base::failure);
+
+    // Serialize into r-value
```

Comment:
In commit "streams: Accept URef obj for VectorReader unserialize" (faadb43546d7befc2a28d5bffa461efc26103cbd)

s/Serialize/Deserialize

Reply:
Thanks, took your suggested test case